pkgs/development/compilers/flutter/default.nix

```diff
@@ -31,19 +42,30 @@ in
     pname = "flutter";
     version = "3.3.8";
     dartVersion = "2.18.4";
-    hash = "sha256-QH+10F6a0XYEvBetiAi45Sfy7WTdVZ1i8VOO4JuSI24=";
+    flutterHash = {
+      linux = "sha256-QH+10F6a0XYEvBetiAi45Sfy7WTdVZ1i8VOO4JuSI24=";
```

Comment:
Same as here: https://github.com/NixOS/nixpkgs/pull/196133/files#r1013277752

Reply:
yes, now I understand where is the issue - I broke linux builds.

Reply:
But I cannot understand how. I've tried to build `flutter` & `flutter2` on  cd09fd5 commit on linux virtual machine and it builds ok. Can you show how and where CI fails?

Reply:
@thiagokokada ?

Reply:
I was not talking about the CI, it seems ok to me at least.

Reply:
@tanshihaj @thiagokokada Not sure I understand correctly, but I created PR, which may resolves this discussion for supporting both aarch and x86 based on this PR. Could you take a look? 

https://github.com/NixOS/nixpkgs/pull/210067